concepts/05 UI Components/TreeList/00 Getting Started with TreeList/10 Customize Columns/5 Resize Columns.md

```diff
@@ -0,0 +1,124 @@
+TreeList columns have equal widths by default. You can set each column's [width](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#width) or indicate that all columns should adjust their widths to their content ([columnAutoWidth](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#columnAutoWidth)). Users can resize columns if you enable the [allowColumnResizing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#allowColumnResizing) property.
```

Comment:
`TreeList columns have equal widths by default.`

We should rewrite this sentence without using "by default". Can we say the following instead?

`TreeList columns have equal widths (the default setting).`

Reply:
Applied this.

Reply:
Thank you.